hack/jenkins/gotest-dockerized.sh

```diff
@@ -45,5 +45,5 @@ docker run --rm=true \
   -e "KUBE_VERIFY_GIT_BRANCH=${KUBE_VERIFY_GIT_BRANCH:-}" \
   -e "REPO_DIR=${REPO_DIR}" \
   -e "HOST_ARTIFACTS_DIR=${HOST_ARTIFACTS_DIR}" \
-  -i gcr.io/google_containers/kubekins-test:v20160822 \
+  -i gcr.io/k8s-testimages/kubekins-test:1.5-latest \
```

Comment:
Let's use tag 1.5-v20161007-49d47fe

Since the latest can change everytime we push a new image.